test/test_region.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+
+client = TestClient(app)
+
+
+@pytest.fixture
+def client():
+    return TestClient(app)
```

Comment:
Fixtures should be in `conftest.py`

Reply:
Also, you should create TestClient in context manager like Denis does. Basically, you can pull and merge his changes into your branch and you'll just use his solution

Reply:
You have to use context manager while TestClient creation, otherwise events will not work

Reply:
+1